canadarm/worlds/simple.world

```diff
@@ -47,7 +47,7 @@
           <visual name='visual'>
             <geometry>
               <mesh>
-                <uri>/home/spaceros-user/demos_ws/install/simulation/share/simulation/models/canadarm/meshes/earth.dae</uri>
+                <uri>model://canadarm/meshes/earth.dae</uri>
```

Comment:
As far as I recall, these were hard-coded to get the moveit_config working which has to be refactored anyway so I won't block on this.